diracx-routers/src/diracx/routers/auth/utils.py

```diff
@@ -73,7 +73,7 @@ class GrantType(StrEnum):
 
     authorization_code = "authorization_code"
     device_code = "urn:ietf:params:oauth:grant-type:device_code"
-    refresh_token = "refresh_token"
+    refresh_token = "refresh_token"  # noqa: S105   # False positif of Bandit about hard coded password
```

Comment:
either fix the typo or remove the comment

```suggestion
    refresh_token = "refresh_token"  # noqa: S105
```